less/common/mixins/extension-styles.less

```diff
@@ -0,0 +1,7 @@
+.if-extension-enabled(@extension) {
+  // Placeholder mixin
```

Comment:
Does this still need an implementation?

Reply:
No. If no extension defines this, the Less compilation will error, so we need to make sure it is defined at least once.